pkgs/by-name/re/resticprofile/package.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+}:
```

[Nixpkgs convention](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax):
```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
, installShellFiles
}:
```

---

I am using the new [`nixfmt`](https://github.com/NixOS/nixfmt) implementation avaliable as `nixfmt-rfc-style`, which will be the new default for the entire repo in the future. I've been using it in new PRs so that when they format the entire repository I wont get unexpected changes.